Added docs description on changing from Internet-facing to Internal a…

snoby opened the pull request:
Created an addition to the docs for how to change the API ELB from an Internet-Facing to an Internal ELB.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.kubernetes.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.kubernetes.io/reviews/kubernetes/kops/2456)
<!-- Reviewable:end -->

snoby commented:
@justinsb - some more docs for you.

chrislovecnm commented:
@k8s-bot ok to test

chrislovecnm commented on docs/topology.md:
````diff
@@ -43,3 +43,27 @@ than `kubenet`.  Currently the supported options are:
+- change the api loadbalanver type from: Public to Internal... should look like this when done:
+```
+ spec:
+ 11   api:
````
Can we remove the line numbers and add yaml formatting please?

snoby replied:
No problem will do.

chrislovecnm commented on docs/topology.md:
```diff
@@ -43,3 +43,27 @@ than `kubenet`.  Currently the supported options are:
+
+### Steps to change the ELB from Internet-Facing to Internal
+- edit the cluster `kops edit cluster $NAME`
+- change the api loadbalanver type from: Public to Internal... should look like this when done:
```
Should we add a note that kops replace will do this? We can add an issue rather than making the pr change.

geojaz replied:
spelling: API loadbalancer?

geojaz replied:
spelling: API loadbalancer?

snoby replied:
@chrislovecnm - I don't know how kops replace would do this, I'm happy to give it a try however what I've documented works for sure.
@geojaz - will be fixed in my next commit.

chrislovecnm commented:
Thanks for the docs, this is awesome!!! Just one tweak ...

geojaz commented on docs/topology.md:
```diff
@@ -43,3 +43,27 @@ than `kubenet`.  Currently the supported options are:
+ - BEFORE DOING the same command with the `yes` option go into aws and DELETE the api ELB!!!!!!
+ - now run `kops update cluster $NAME --yes`
+ - Finally you will need to do a rolling update so that the instances register wiht the new internal ELB  run 'kops rolling update --cloudonly --force'
+ we have to do cloudonly because we deleted the api ELB so there is no way to talk to the cluster.  The force option is there because kops / terraform doesn't know that we need to update the intances with the ELB so we have to force it.
```
instances?

snoby commented:
I fixed some typos and changed a little bit of grammar. @chrislovecnm - I'm not sure about the `kops replace` command.  I've never tried it in this situation, but I know the method I documented works.

snoby commented:
bump

justinsb commented:
Thanks @snoby
